downloads/plotting.html

```diff
@@ -3,6 +3,10 @@
 title:  Plotting in Julia
 ---
 
+<head>
+  <meta name="description" content="Get started with plotting packages for Julia today. Learn more about plotting in Julia with these powerful external packages here."/>
```

Comment:
wording seems weird imo

Reply:
Hmmm... I'm not all that good at writing descriptions. Maybe someone else can do this one?